src/nreplController.ts

```diff
@@ -46,13 +46,19 @@ const start = (): Promise<CljConnectionInformation> => {
         });
 
         nreplProcess.stderr.on('data', data => {
+            console.info('nrepl stderr =>', data.toString());
```

Comment:
Looks good. Not blocking for now, but just a thought: we could wire this up to a vscode.OutputChannel, so users can see any output without having to run the extension under debugging mode (I think console.info only shows up when debugging the extension?)

But probably not worth doing in this review, probably needs a bit more thought.